assets/js/pages/article/read/ReadArticle.vue

```diff
@@ -1,10 +1,10 @@
 <template>
     <div>
+        <article-title>{{ currentArticleMetadata.title }}</article-title>
```

Comment:
Why not putting that just before <article-content/> ? It seems more logical. And maybe we should display that only if there actually is a title in the metadata